lib/results.js

```diff
@@ -16,6 +16,10 @@ var nextTick = typeof setImmediate !== 'undefined'
 module.exports = Results;
 inherits(Results, EventEmitter);
 
+function coalesceWhiteSpaces(str) {
+    return str.toString().replace(/\s+/g, ' ');
```

Comment:
```suggestion
    return String(str).replace(/\s+/g, ' ');
```
also better than .toString :-)

any reason not to only replace newlines?

Reply:
> any reason not to only replace newlines?

I just copied it from https://github.com/substack/tape/blob/master/lib/results.js#L145 It felt like a sane thing to do.

Reply:
Fair enough!